src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArray = date.split(fromFormat[fromFormat.length - 1]);
+  // eslint-disable-next-line max-len
+  const object = Object.fromEntries(fromFormat.slice(0, -1).map((format, index) => [format, dateArray[index]]));
```

Comment:
Something like that)
```suggestion
  const object = Object.fromEntries(fromFormat.slice(0, -1).map(
    (format, index) => [format, dateArray[index]])
  );
```